package-lock.json

```diff
@@ -56,21 +56,80 @@
           "integrity": "sha512-QjVjwdXIt408MIiAqCX4oUKsgU2EqAGzs2Ppkm4aQYbjm+ZEWEcW4SfFNTr4uMNZma0ey4f5lgLrkB0aX0QMow==",
           "dev": true,
           "requires": {
-            "has-flag": "^3.0.0"
+            "has-flag": "3.0.0"
           }
         }
       }
     },
     "@okta/openapi": {
-      "version": "0.13.0",
-      "resolved": "https://registry.npmjs.org/@okta/openapi/-/openapi-0.13.0.tgz",
-      "integrity": "sha1-OaXIyCoAPmUq5B9P52aRGvoFOqg=",
+      "version": "1.11.2",
+      "resolved": "https://registry.npmjs.org/@okta/openapi/-/openapi-1.11.2.tgz",
+      "integrity": "sha512-sQ1katXkeXHg9X9yzf672XUE0gi5RXCf9e92wi86mc8C1So50cM2mF5WWxr1wLEp+Fkznn8I/0pd8Sa5RgPVIw==",
       "dev": true,
       "requires": {
         "commander": "2.9.0",
         "fs-extra": "3.0.1",
-        "handlebars": "4.0.8",
-        "json-stable-stringify": "1.0.1"
+        "handlebars": "4.4.3",
```

Comment:
yas

Reply:
Didn't touch any packages in yarn.  think this is just from doing `yarn install` after cloning